### PR TITLE
Backport release script fixes to 2.11.x

### PR DESCRIPTION
**Pull request by lrytz**
Backports a number of fixes to the release script that were made during the 2.12.0-M1 release.

### CODE REVIEW

**Comment by lrytz** [comment]
Review by @retronym

File: versions.properties [hunk 1]
```diff
@@ -3,32 +3,40 @@
...
 
 # external modules shipped with distribution, as specified by scala-library-all's pom
-scala-xml.version.number=1.0.3
-scala-parser-combinators.version.number=1.0.3
+scala-xml.version.number=1.0.4
+scala-parser-combinators.version.number=1.0.4
 scala-continuations-plugin.version.number=1.0.2
 scala-continuations-library.version.number=1.0.2
-scala-swing.version.number=1.0.1
-akka-actor.version.number=2.3.4
```
Comment (retronym) [inline versions.properties:25]: I wasn't sure what our policy was here (upgrade akka or not?) I checked the history of this file and found that we have bumped the version twice in the 2.11.x series: ef427b27a / 103c9850c, so I think this is fine.

**Comment by retronym** [comment]
LGTM

**Pull request merged**